<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>crashmeClass</class>
 <widget class="QMainWindow" name="crashmeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>crashme</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="crashTypeLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Capture it</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Crash Type</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>231</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="boradioButton">
       <property name="text">
        <string>buffer overflow</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="horadioButton">
       <property name="text">
        <string>heap overflow</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="umradioButton">
       <property name="text">
        <string>uninitialized memory access</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="zpradioButton">
       <property name="text">
        <string>zero page access</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="crashButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>28 Days Later</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Crash me!</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="descriptionTextBrowser">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>256</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>547</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="crashme.qrc"/>
 </resources>
 <connections/>
</ui>
